karabo_data/tests/mockdata/detectors.py

```diff
@@ -17,19 +17,26 @@ class DetectorModule:
         'trailer',
     ]
 
-    def __init__(self, device_id, frames_per_train=64):
+    def __init__(self, device_id, frames_per_train=64, raw=True):
         self.device_id = device_id
         self.frames_per_train = frames_per_train
+        if not raw:
+            self.image_dims = self.image_dims[1:]
```

Comment:
Maybe add a comment about this - I think I know why it's needed, but it's a bit mysterious.

Reply:
Okidoke